app/services/chat_channels/create_with_user.rb

```diff
@@ -0,0 +1,42 @@
+module ChatChannels
+  class CreateWithUser
+    def initialize(options = {})
+      @users = options.fetch(:users)
+      @channel_type = options.fetch(:channel_type, "direct")
+      @contrived_name = options.fetch(:contrived_name, "New Channel")
+      @membership_role = options.fetch(:membership_role, "member")
+    end
```

Comment:
I would replace this with `def initialize(users: [], channel_type: "direct", contrived_name: "New Channel", "membership_role: "member"`).

It's much clearer what the method requires